### ISSUE TITLE
are there any plans to make a navigation menu?

### ISSUE THREAD

**Issue by csbenjamin** [angular/material#3040]
In this website http://www.google.com/design there is a horizontal nav on top of the page. Is there some plan to make a navigation menu like that?

**Comment by Anthropic** [comment]
Could but wrong but I believe that is a combination of these two:
https://material.angularjs.org/HEAD/#/demo/material.components.tabs
https://material.angularjs.org/HEAD/#/demo/material.components.toolbar

**Comment by csbenjamin** [comment]
Well, I'm doing something like that. Every time that the url changes, the controller changes the md-selected attribute and when a user clicks in one tab, the controller changes the url. But there are much html generated to something simple and if the current url doesn't match with any menu item, we need the possibility to have none tab active.

A question: How do I align the tabs on the right instead on left? I put `float: right` in md-tab-item and the tabs disappeared.

OBS: My english is not so good

**Comment by csbenjamin** [comment]
I'm thinking to use the tab directive and make a md-nav directive from it

**Comment by vanntian** [comment]
next week https://github.com/angular/material/issues/78#issuecomment-106285515

**Comment by janesconference** [comment]
@vanntian @csbenjamin #78 s not the same thing as this issue.
@csbenjamin is actually asking for a nav toolbar similar to this site's: http://www.google.com/design

I think the problem arises from the 'menu' term.

**Comment by csbenjamin** [comment]
@vanntian you're right, it is not the same thing. I closed this issue because there is not a nav toolbar similar to http://www.google.com/design in google design spec. But I'm reopening this issue and If anyone thinks that angular-material should not have a nav toolbar similar to http://www.google.com/design, close this issue please.

**Comment by janesconference** [comment]
BTW, @csbenjamin, material-angular has a toolbar that scroll-shrinks (see https://material.angularjs.org/latest/#/demo/material.components.toolbar, in the "scroll-shrink" section). I guess it's just a matter of taking that toolbar and adjusting the CSS. But I never tried, so no guarantees.

**Comment by csbenjamin** [comment]
@janesconference The scroll-shrinks is cool, but what I really need is the "ARTICLES|RESOURCES|EVENTS|JOBS|ABOUT" part in the nav toolbar in the site http://www.google.com/design

**Comment by rschmukler** [comment]
@csbenjamin since there isn't really any functionality exposed by the nav menu, we view a component as overkill. Really it's just some CSS that needs to be applied to a `ul` or `nav` or whatever you want around some `a` elements.

**Comment by csbenjamin** [comment]
@rschmukler  It is more than just some CSS. The nav menu must interact with $location service to know if each menu item is active or not. And must have some animation when we change from one item active to other.

**Comment by janesconference** [comment]
You could always use onClick and change location or directly an ui-sref, if you're using ui-router.

**Comment by csbenjamin** [comment]
@janesconference I think something like the tabs, with a bar below the active menu item and when we change the route, there will be some animation changing the bar to the next active menu item. It would be a simple component, but very useful.

**Comment by csbenjamin** [comment]
also, when we have so many menu item, we would have some pagination, like the tabs component

**Comment by nistala** [comment]
+1 would like to see a top menu similar to what is on the google design home page with $location support.
![screenshot from 2015-08-30 13 26 09](https://cloud.githubusercontent.com/assets/4357428/9569268/b935a106-4f1a-11e5-973e-40f475ef8ef6.png)

**Comment by jma7889** [comment]
@rschmukler A navigation menu is an essential feature no matter how simple the implementation will be. It is not an overkill! Isn't it a good thing to help others while you don't have to do much?

**Comment by super-coder** [comment]
so ELI5 how do I do a website with 10-20 pages?  What is the alternative to [Bootstrap's navbar](http://getbootstrap.com/examples/navbar/)? Naturally I want some solution that works on all devices and when resizing

**Comment by ThomasBurleson** [comment]
Please post this in the [Angular Material Forum](https://groups.google.com/forum/#!forum/ngmaterial)
